doc/changes.md

```diff
@@ -1,4 +1,133 @@
 # Changelog
+## Release v0.4.0 (2021/06/20)
+
+[![Digital Object Identifier for PyGMT v0.4.0](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.4978645)
+
+### Highlights
+
+* 🎉 **Fourth minor release of PyGMT** 🎉
+* 10 new PyGMT functions and methods
+* New and improved gallery examples and tutorials
+* Drop GMT 6.1.1 support
+
+### New Features
+
+* Wrap grdlandmask ([#1273](https://github.com/GenericMappingTools/pygmt/pull/1273))
+* Wrap grdgradient ([#1269](https://github.com/GenericMappingTools/pygmt/pull/1269))
+* Wrap grdfill ([#1276](https://github.com/GenericMappingTools/pygmt/pull/1276))
+* Wrap grdclip ([#1261](https://github.com/GenericMappingTools/pygmt/pull/1261))
+* Handle geopandas and shapely geometries via geo_interface link ([#1000](https://github.com/GenericMappingTools/pygmt/pull/1000))
+* Wrap velo ([#525](https://github.com/GenericMappingTools/pygmt/pull/525))
+* Wrap histogram ([#1072](https://github.com/GenericMappingTools/pygmt/pull/1072))
+* Wrap wiggle ([#1145](https://github.com/GenericMappingTools/pygmt/pull/1145))
+* Support passing string type numbers, geographic coordinates and datetimes ([#975](https://github.com/GenericMappingTools/pygmt/pull/975))
+* Wrap rose ([#794](https://github.com/GenericMappingTools/pygmt/pull/794))
+* Wrap solar ([#804](https://github.com/GenericMappingTools/pygmt/pull/804))
+* Wrap blockmean ([#1092](https://github.com/GenericMappingTools/pygmt/pull/1092))
+* Allow load_earth_relief() to load the original land-only 01s or 03s SRTM tiles ([#976](https://github.com/GenericMappingTools/pygmt/pull/976))
```

Comment:
Sort the entries so that the "Wrap ..." are put together.

Reply:
Sorting all done in https://github.com/GenericMappingTools/pygmt/pull/1341/commits/430e7d96992d757c18234c077903611e7e0c8562